src/main/java/com/salesforce/dev/framework/DataDrivenManager.java

```diff
@@ -49,4 +51,92 @@ public DataDrivenManager(){
         return accountsArray.iterator();
     }
 
+    public Iterator<ViewSalesForce[]> getDataView(String fileJson){
+
+        Collection<ViewSalesForce[]> viewSalesForcesArray =new ArrayList<ViewSalesForce[]>();
+        try {
+            parser = new JSONParser();
+            String pathFileJson = "src\\test\\resources\\" + fileJson;
+            Object jsonObject = parser.parse(new FileReader(pathFileJson));
+            JSONArray jsonArray = (JSONArray) jsonObject;
+
+            ObjectMapper objectMapper = new ObjectMapper();
+
+            List<ViewSalesForce> navigation = objectMapper.readValue(jsonArray.toJSONString(),
+                    objectMapper.getTypeFactory().constructCollectionType(
+                            List.class, ViewSalesForce.class));
+
+            for (int i = 0 ; i< navigation.size(); i++){
+                ViewSalesForce viewSalesForce = (ViewSalesForce)navigation.get(i);
+                viewSalesForcesArray.add(new ViewSalesForce[]{viewSalesForce});
+            }
+
+        } catch (FileNotFoundException e) {
+            e.printStackTrace();
+        } catch (IOException e) {
+            e.printStackTrace();
+        } catch (ParseException e) {
+            e.printStackTrace();
+        }
+        return viewSalesForcesArray.iterator();
+    }
+
+    public Iterator<ViewSalesForce[]> getDataViewtoEdit(String fileJson){
+
+        Collection<ViewSalesForce[]> viewSalesForcesArray =new ArrayList<ViewSalesForce[]>();
+        try {
+            parser = new JSONParser();
+            String pathFileJson = "src\\test\\resources\\" + fileJson;
+            Object jsonObject = parser.parse(new FileReader(pathFileJson));
+            JSONArray jsonArray = (JSONArray) jsonObject;
+
+            ObjectMapper objectMapper = new ObjectMapper();
+
+            List<ViewSalesForce> navigation = objectMapper.readValue(jsonArray.toJSONString(),
+                    objectMapper.getTypeFactory().constructCollectionType(
+                            List.class, ViewSalesForce.class));
+
+            for (int i = 0 ; i< navigation.size(); i++){
+                ViewSalesForce viewSalesForce = (ViewSalesForce)navigation.get(i);
+                viewSalesForcesArray.add(new ViewSalesForce[]{viewSalesForce});
+            }
+
+        } catch (FileNotFoundException e) {
+            e.printStackTrace();
+        } catch (IOException e) {
+            e.printStackTrace();
+        } catch (ParseException e) {
+            e.printStackTrace();
+        }
+        return viewSalesForcesArray.iterator();
+    }
+
+    public Iterator<Campaign[]> getCampaign(String fileJson) {
+
+        Collection<Campaign[]> campaignsArray = new ArrayList<Campaign[]>();
+        try {
+            parser = new JSONParser();
+            Object jsonObject = parser.parse(new FileReader("src\\test\\resources\\" + fileJson));
```

Comment:
Try to use absolute paths in order to avoid problems when executing on Windows, Linux and MacOS